aws/resource_aws_lb_target_group.go

```diff
@@ -83,6 +83,12 @@ func resourceAwsLbTargetGroup() *schema.Resource {
 				ValidateFunc: validation.IntBetween(0, 3600),
 			},
 
+			"proxy_protocol_v2": {
```

Comment:
Does this attribute support updates? If so, we should add an acceptance test step for that. If not, it should be marked as `ForceNew: true,`